x/did/internal/types/msgs.go

```diff
@@ -108,17 +109,17 @@ func (msg MsgAddCredential) String() string {
 		string(msg.DidCredential.Claim.Id), msg.DidCredential.CredType, string(msg.DidCredential.Issuer))
 }
 
-func (msg MsgAddCredential) ValidateBasic() sdk.Error {
+func (msg MsgAddCredential) ValidateBasic() error {
 	// Check if empty
 	if strings.TrimSpace(msg.DidCredential.Claim.Id) == "" {
-		return ErrorInvalidDid(DefaultCodespace, "claim id should not be empty")
+		return sdkerrors.Wrap(ErrInvalidClaimId, "claim ID not be empty")
```

Comment:
"claim ID **should** not be empty"

Reply:
Yep, I don't even know why that was changed